src/main/public/js/double-click-preventer.js

```diff
@@ -0,0 +1,14 @@
+(function () {
+  document.addEventListener('DOMContentLoaded', function (event) {
+
+    var submitType = document.querySelector('input[type=submit]')
+
+    if (submitType != null) {
+      submitType.addEventListener('dblclick', function (event) {
```

Comment:
While this story says preventing dblclick what we're really trying to achieve is not sending multiple requests to the backend, I think a click event would be better and then disabling the input

Reply:
I agree. I can imagine that with backend responding slow user would be able to fire two separate clicks that are not handled by double click event. The ultimate goal is to prevent multiple same requests being fired to backend.

Reply:
how is it not covered? 
would be worth checking IE with the above solution, I would have thought that as long as you disable the input after the first click it would submit fine?

Reply:
Thanks all for the inputs. In conversation with @robertparkinson and GDS pattern we tends to use javascript first before using JQuery (where we have to maintain different version due to different supported browsers). We also discussed disabling submit button which may not go down well with UX. Testing other review comments but we can check with UX on this.